UEBPrep/app/models/user.rb

```diff
@@ -62,6 +62,7 @@ def self.find_for_twitter_oauth(auth, signed_in_resource=nil)
 
         user = User.create(provider:auth.provider,
                            uid:auth.uid,
+                           username:auth.uid+"@twitter.com",
```

Comment:
Also I believe we talked about not allowing special characters in usernames, only numbers and letters.